bounties/npm/node-prompt-here/1/README.md

```diff
@@ -0,0 +1,13 @@
+# Overview
+
+`node-prompt-here` is a package to open a console window at given absolute directory, this package are vulnerable to Command Injection.
+
+The `runCommand()` is called by `getDevices()` function in file `linux/manager.js`, which is required by the index. `process.env.NM_CLI` in the file `linux/manager.js` . This function is used to construct the argument of function `execSync()`, which can be controlled by users without any sanitization.
```

Comment:
Description is wrong, please update it.

The `open()` function contains a `vulnerable lines of code` (https://github.com/s-a/node-prompt-here/blob/master/index.js#L19) which concatenates insecure `user-supplied` inputs without any check, leading to `RCE`.

Cheers,
Mik

Reply:
Thanks for looking into this further and I have updated the `README.md` with your suggested update.

Much appreciated ❤️